appveyor.yml

```diff
@@ -5,9 +5,6 @@ version: 62.0.0-CI{build}
 
 shallow_clone: true
 
-# Start builds on tags only (GitHub and BitBucket)
-skip_non_tags: true
```

Comment:
Is there a reason you've removed this? How are you going to handle the version conflicts?

Reply:
Thanks for asking! Yes, there is a reason, it was discussed in #56.

(Short answer: We want to always run the AppVeyor builds on each PR, to ensure that the project still builds with each change. Ideally, the MyGet files should only be pushed whenever a tag had been added, though. I think this can be done, but I'm not 100% sure of how yet.)

Will revert this part of the change since, as you say, it is not a "formatting change" only.